From f7b20fc6aa572fc68215e535775606c7b17f6ed4 Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Fri, 1 Jun 2018 16:25:02 +0800
Subject: [PATCH 3898/5242] MLK-18477-4 arm64: dts: fsl-imx8qm-mek-hdmi: Don't
 assign dpu disp clk parent

commit  430598dfb79b15e1caeee3184b60789bbd241a57 from
https://source.codeaurora.org/external/imx/linux-imx.git

The dpu driver may properly handle display clock parent selection now.
Thus, let's remove the assigned-clock and assigned-clock-parent
device tree properties from the dpu device tree node.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-mek-hdmi.dts     |    9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-hdmi.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-hdmi.dts
index e9fe343..4ef6af4 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-hdmi.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-hdmi.dts
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 NXP
+ * Copyright 2017-2018 NXP
  *
  * This program is free software; you can redistribute it and/or
  * modify it under the terms of the GNU General Public License
@@ -48,13 +48,6 @@
 	};
 };
 
-&dpu1 {
-		assigned-clocks = <&clk IMX8QM_DC0_DISP0_SEL>,
-						<&clk IMX8QM_DC0_DISP1_SEL>;
-		assigned-clock-parents = <&clk IMX8QM_DC0_BYPASS_0_DIV>,
-						<&clk IMX8QM_DC0_PLL1_CLK>;
-};
-
 &ldb1_phy {
 	status = "disabled";
 };
-- 
1.7.9.5

